app/assets/stylesheets/forever_style_guide/modules/_modals.scss

```diff
@@ -0,0 +1,57 @@
+// preventing  .style-guide-partial .style-guide-partial-rendered > * from forcing modal to render inline 
+.style-guide-partial .style-guide-partial-rendered .modal {
```

Comment:
considering that this is basically a tidying up for style guide specific rendering, perhaps it would better belong as its own module under style_guide modules?